cmd/http/main.go

```diff
@@ -82,6 +84,10 @@ func main() {
 	}
 	hs := httpserver.New(ctx, cancel, svcName, httpServerConfig, api.MakeHandler(svc, tracer, logger), logger)
 
+	homeSvc := client.New(svcName, cfg.MFRelease, logger, cancel)
```

Comment:
There is repetition of these lines in every `main()`. Didn't @arvindh123 unify our main functions to avoid repetition?